app/views/layouts/_header.html.haml

```diff
@@ -10,4 +10,4 @@
       .navbar-text.pull-right
         = succeed "." do
           %strong= current_user.name
-        = link_to "Logout", destroy_user_session_path, method: :delete, class: 'navbar-link'
\ No newline at end of file
+        = link_to(raw('<span class="glyphicon glyphicon-off"></span> Logout'), destroy_user_session_path, method: :delete, class: 'navbar-link')
```

Comment:
That would be `<i class='fa fa-power-off'></i> Log Out` or something similar....